### PR TITLE
Margin changes

### LINKED ISSUE #961: The grid doesn't have the spaces defined
## Describe the bug
The layout of first section doesn't have the spaces defined for the grid in [Mockup S3](https://www.figma.com/file/KEXxItd8EloJGbXswuUDqq/Mockup---S3-(License-Language-Changes)?node-id=199%3A354&viewport=969%2C732%2C0.13086995482444763). 

## Expected behavior
Use the grid and the columns defined for placing elements.

## Screenshots
layout grid

### PR DESCRIPTION
**Pull request by akanksha-v**
Fixes #961 by @panchovm 

## Description
I have made certain changes related to margin to match the spaces defined for grid in Mockup S3.

## Screenshots
margin-changes

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/ImageDetails/PhotoDetails.vue [hunk 1]
```diff
@@ -31,8 +31,8 @@
                            :providerName="providerName" />
     </div>
     </div>
-    <div class="column image-info">
-      <div class="margin-vertical-normal">
+    <div class="column image-info" id="right-side">
```
Comment (brenoferreira) [inline src/components/ImageDetails/PhotoDetails.vue:34]: Please use the Vocabulary margin classes https://cc-vocabulary.netlify.app/?path=/docs/tokens-spacing--figma#usage

Comment (akanksha-v) [reply src/components/ImageDetails/PhotoDetails.vue]: @brenoferreira Thank you for reviewing my pr. I have updated the suggested changes, thanks.

**Comment by zackkrida** [comment]
Hi @akanksha-v once you fix the merge conflicts I can close this pull request! Looks good. If it's easier for you to make a new PR, that's okay too. Thanks!

**Comment by akanksha-v** [comment]
@zackkrida Sorry for the inconvenience and thank you for reviewing my pr. I have created another pr #1044 for the same issue 
 resolving its merge conflicts

**Pull request closed without merging**